components/molecules/EventCard/EventCardMax.js

```diff
@@ -72,8 +68,8 @@ const EventCardMax = ({ toggle, setFav, isFav, el, description }) => {
             ? <p>{description.description}</p>
             : <p>There is no description for this event</p>}
       </div>
-      <div onClick={(e) => pushLink(e, el.scrapedEventLink)} id="button">
-        <Button medium background={colors.primary}>Visit Source</Button>
+      <div id="button">
+        <Button medium background={colors.primary}><a style={{color: 'inherit'}} href={el.scrapedEventLink} target="_blank" rel="noreferrer noopener">Visit Source</a></Button>
```

Comment:
Kindly put the `a` tag, text in `a` tag, closing `a` tag and closing button component on a new line. @Oluwa-Femi

Reply:
@twumm Thanks. Resolved